packages/klevu-ui/src/components/klevu-search-landing-page/klevu-search-landing-page.tsx

```diff
@@ -138,7 +147,15 @@ export class KlevuSearchLandingPage {
       if (this.usePersonalisation === undefined && settings?.klevu_uc_userOptions.enablePersonalisationInSearch) {
         this.usePersonalisation = true
       }
-      if (this.showSearch === undefined) this.showSearch = settings?.klevu_uc_userOptions.showSearchBoxOnLandingPage
+      if (this.showSearch === undefined) {
+        this.showSearch = settings.klevu_uc_userOptions.showSearchBoxOnLandingPage
+      }
+      if (this.showFilters === undefined) {
```

Comment:
I think this is one of those cases where we should have filters shown as default. So let's rename `showFilters` to `hideFilters` and then invert the logic `this.hideFilters = !settings?.klevu_filtersEnabled`

Reply:
changed